docker/testdata/BUILD

```diff
@@ -246,6 +251,15 @@ docker_build(
     },
 )
 
+docker_push(
+    name = "push_stamp",
+    image = ":link_with_files_base",
+    registry = "gcr.io",
+    repository = "convoy-adapter/bazel-test",
```

Comment:
@samuraisam's note from #54: it might be worth putting a stamped value in here too. (it seems to work, but it'd be good to test.)